run.sh

```diff
@@ -197,13 +197,27 @@ check_themes() {
       h3 "($i/$theme_count) '$theme_name' not found. Installing"
       WP theme install --quiet "$theme_name"
       STATUS $?
+      ((i++))
+      continue
+    fi
+
+    # Locally volumed themes
+    if [[ $theme_name =~ ^\[local\] ]]; then
+      themes["${theme_name##*]}"]="${theme_name##*]}"
+      h3 "($i/$theme_count) '${theme_name##*]}' listed as a local volume. SKIPPING..."
+      STATUS SKIP
+      ((i++))
       continue
     fi
 
     # If $theme_name matches a URL using the new format => set $theme_name & $theme_url
+    # If url is from github, correct for the branch name in the plugin name
     if [[ $theme_name =~ ^\[.+\]https?://[www]?.+ ]]; then
       theme_url=${theme_name##\[*\]}
       theme_name="$(echo "$theme_name" | grep -oP '\[\K(.+)(?=\])')"
+      if [[ $theme_url =~ github.com ]]; then
```

Comment:
The URL for plugins installed from github.com is appended with the branch name. This corrects for that so that the build can still check if those plugins are installed or not.

In other words: If I'm installing `akismet` from the master branch of the akismet repo, the url will look like this: `akismet-master`. This fixes it to look like this: `akismet`.